flags.go

```diff
@@ -341,6 +344,67 @@ type flagStorage struct {
 	DebugMutex      bool
 }
 
+const GCSFUSE_PARENT_PROCESS_DIR = "gcsfuse-parent-process-dir"
+
+// (1) returns the same filepath in case of absolute path or empty filename.
+// (2) for relative path with . && .. or nothing, it resolves smartly the path
+// in case of child proces it resolves with respect to GCSFUSE_PARENT_PROCESS_DIR
+// otherwise with respect to the current working dir.
+// (3) for relative path starting with ~, it resolves with respect to home dir.
+func getResolvedPath(filePath string) (resolvedPath string, err error) {
+	if filePath == "" || path.IsAbs(filePath) {
+		resolvedPath = filePath
+		return resolvedPath, err
+	}
+
+	// relative path starting with tilda (~)
+	if strings.HasPrefix(filePath, "~") {
+		homeDir, err := os.UserHomeDir()
+		if err != nil {
+			return "", fmt.Errorf("Fetch home dir: [%w]", homeDir)
+		}
+		return filepath.Join(homeDir, filePath[2:]), err
```

Comment:
Should you check for HasPrefix("~/") above since you are omitting first two characters?

Reply:
Yeah. Done.